### ISSUE TITLE
Slider - tab order should be reverted

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#8272]
**Describe the problem**
PR #8190 should be reverted based on discussion in https://github.com/patternfly/patternfly/pull/5172 and https://github.com/patternfly/patternfly/pull/5157
